MediaPushPlugin/README.md

```diff
@@ -55,21 +55,21 @@ Media Push Plugin have REST API to control the plugin.
 
 Call the REST Method below to let Ant Media Server broadcast the web page. You should pass the url of the web page and can pass streamId as query parameter you wanted to use as a parameter.
    ```
-   curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "https://<ant-media-server-domain>/<your-webapp-name>/rest/v1/media-push/start" -d '{"url": "http://example.com", "width": 1280, "height": 720}'
+   curl -i -X POST -H "Accept: Application/json" -H "Content-Type: application/json" "https://<ant-media-server-domain>/<your-webapp-name>/rest/v2/media-push/start" -d '{"url": "http://example.com", "width": 1280, "height": 720}'
```

Comment:
Please use REST naming conventions
https://nordicapis.com/10-best-practices-for-naming-api-endpoints/

>RESTful URIs should refer to a resource that is a thing (noun) instead of referring to an action (verb) because nouns have properties which verbs do not have – similar to resources have attributes.

Reply:
It's reverted